2.x/src/main/resources/archetype-resources/api/src/main/java/api/impl/__D-moduleClassPrefix__ServiceImpl.java

```diff
@@ -34,7 +34,7 @@ public void setDao(${D-moduleClassPrefix}DAO dao) {
     /**
      * @return the dao
      */
-    public ${D-moduleClassPrefix}DAO getDao() {
+    private ${D-moduleClassPrefix}DAO dao;private ${D-moduleClassPrefix}DAO dao;{

```

Comment:
Remove the second call **dao;private ${D-moduleClassPrefix}DAO dao;**, Also dont get rid of getDao, dont change it

Reply:
i would not remove getDao. change from dao to getDao, You will also this time learn how to squash your commits into one, Squashing commit is another way of adding your commits into one such that it can be very easy for someone to review. you can follow this link https://wiki.openmrs.org/display/docs/Git-Squashing+commits

Reply:
Can you try adding getDao instead of dao

Reply:
Can you add getDao instead of dao